tests/nnstreamer_sink/unittest_sink.cc

```diff
@@ -5845,14 +5848,18 @@ TEST (tensorFilterCustomEasy, inCodeFunc01)
 TEST (tensorFilterCustomEasy, unregister1_p)
 {
   int ret;
-  const GstTensorsInfo info_in = {
-    .num_tensors = 1U,
-    .info = { { .name = NULL, .type = _NNS_UINT8, .dimension = { 1, 10, 1, 1 } } },
-  };
-  const GstTensorsInfo info_out = {
-    .num_tensors = 1U,
-    .info = { { .name = NULL, .type = _NNS_UINT8, .dimension = { 1, 10, 1, 1 } } },
-  };
+  GstTensorsInfo info_in = { 0, };
+  GstTensorsInfo info_out = { 0, };
+
+  info_in.num_tensors = 1U;
```

Comment:
let's use gst_tensors_info_init() for later work.

Reply:
Updated! Thanks.